DashApp/callbacks/gene_callbacks.py

```diff
@@ -392,26 +409,8 @@ def update_comparison_plot(gene1: str, gene2: str, selected_datasets: list,
                 merged_data[col2].max()
             )
             
-            # TODO calculate these before transform data
-            # TODO FROM HERE
             # Add regression line if there are enough data points
             if len(merged_data) > 1:
-                x_values = merged_data[col1].values
-                y_values = merged_data[col2].values
-                
-                #print(f"x_values: {x_values}")
-                #print(f"y_values: {y_values}")
-                
-                # Calculate Pearson correlation
-                pearson_corr, p_value = stats.pearsonr(x_values, y_values)
-                
-                # Calculate Spearman correlation
-                spearman_corr, spearman_p = stats.spearmanr(x_values, y_values)
-
-                # Calculate Pearson correlation for log2(TPM+1)
-                log_x = np.log2(x_values + 1)
-                log_y = np.log2(y_values + 1)
-                pearson_corr_log, p_value_log = stats.pearsonr(log_x, log_y)
                 
                 # Standard regression line calculation
                 slope, intercept, r_value, p_value, std_err = stats.linregress(x_values, y_values)
```

Comment:
But then here `x_values` and `y_values`, which are still holding the untransformed values, are used to calculate the regression line. Just pull these from the dataframe again as they will now have had the transform applied

Reply:
Thanks for fixing